doc/src/learn/sui-glossary.md

```diff
@@ -67,7 +67,7 @@ stage is considered closure for an exchange or other blockchain transaction.
 
 ### Gas
 
-As with other blockchains, [gas](https://www.investopedia.com/terms/g/gas-ethereum.asp) in Sui is the currency for the cost of conducting a transaction.
+As with other blockchains, [gas](https://www.investopedia.com/terms/g/gas-ethereum.asp) refers to the currency for the cost of conducting a transaction. In Sui, the currency is eponymous: Sui
```

Comment:
Should we refer the currency as `SUI` instead of `Sui`?

Reply:
Yep, this is the industry convention: ticker caps for coins.

Reply:
Thank you both. I fixed this here and added in in the separate Sui definition.